server/src/main/java/org/opensearch/indices/replication/SegmentFileTransferHandler.java

```diff
@@ -97,20 +98,26 @@ public MultiChunkTransfer<StoreFileMetadata, FileChunk> createTransfer(
         return new MultiChunkTransfer<>(logger, threadPool.getThreadContext(), listener, maxConcurrentFileChunks, Arrays.asList(files)) {
```

Comment:
@mch2 did a few changes here to fix the tests, could you please take a look? thank you!